akka-cluster-sharding-typed/src/main/scala/akka/cluster/sharding/typed/scaladsl/ClusterSharding.scala

```diff
@@ -253,9 +253,27 @@ object EntityTypeKey {
   /**
    * Send a message to the entity referenced by this EntityRef using *at-most-once*
    * messaging semantics.
+   *
+   * Example usage:
+   * {{{
+   * val target: EntityRef[String] = ...
+   * target tell "Hello"
```

Comment:
We try to avoid operator style method calls except for with symbolical methods, so this should be `target.tell("Hello")`

Reply:
Get it fixed :)